extensions/BMO/lib/Data.pm

```diff
@@ -42,6 +42,8 @@ my $mozreview_url_re = qr{
     $
 }ix;
 
+my $phabricator_uri = Bugzilla->params->{phabricator_base_uri} || 'https://example.com';
```

Comment:
it's not safe to call Bugzilla->params at compile time (outside of a sub in a module).
This will be frozen to the value that was in data/params at httpd startup time, never changing until a deployment happens.

Also this variable appears to be unused.